tests/search/bm25_feature/bm25_feature.rb

```diff
@@ -54,6 +105,33 @@ def assert_scores_for_query(query, exp_scores)
     end
   end
 
+  def use_sdfile(sdfile)
```

Comment:
I don't think this is needed anymore. I think the sd-file can have a different name than the search / document tag inside. Please try without. If it is not possible we should have a common function in search_test.rb instead.

Reply:
[14:01:46.549] Invalid application package: default.default: Error loading model: Search definition file name ('test.0') and name of search element ('test') are not equal for file 'test.0.sd'